admin/admin-strings.php

```diff
@@ -9,8 +9,19 @@
  * @since 1.6
  */
 class PLL_Admin_Strings {
-	protected static $strings = array(); // strings to translate
-	protected static $default_strings; // default strings to register
+	/**
+	 * Stores the strings to translate.
+	 *
+	 * @var array
```

Comment:
`string[]` is more precise than `array`

Reply:
Well in fact it's not an array of strings, but an array of arrays.
See https://github.com/polylang/polylang/blob/20c00ffe116c4f203e5088178c4fede8cd1319e1/admin/admin-strings.php#L48